content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
+
+**Long answer**
+
+RSE can leverage all the above issues and more by accompanying researchers in their software development journey. RSEs' technical knowledge and creativity on solving problems allow seeing where researchers would benefit the most. Here are a few examples/use cases: 
+
+* Researchers have great ideas, but might not know how, or have time to develop the corresponding tools 
+* Help to get rid of repetitive or time consuming steps (that users don't want to do themselves) by introducing automatization 
+* Write documentation so the information doesn't get lost when people leave
+* Make the regular technology updates to prevent that it become obsolete and break
+* Supporting when switching from laptop to compute clusters 
+* Many groups need the same thing, RSEs know how to create flexible templates to simplify e.g. spinning up a website to present research outputs
+* … what other good examples? 
+
+They can also act as intermediate in technical discussions about computing needs, as they know the needs of researchers since they are involved in research in a daily basis, and are also technology-aware people. 
+
+As a summary, researchers with very deep expertise and RSEs with shallower but wider range of interests and skills complement each other very efficiently. 
+
+#### How organisations can help?
+
+Many organisations already have people doin RSE work, but the people and the organisation might not be aware of the term RSE. To give those people a sense of belonging and knowing that there is others like them, organisations could:
+
+1) Support community involvement and self development
+    - Give people room to be active in RSE communities, attend conferences and trainings beyond academic work.
+
+2) Spread the word - be an RSE ambassador
+    - Recognize and highlight RSE (often behind the scenes and non-academic) work
+    - Invite RSEs to talk about their work
+    - Support RSEs building their own network within the organisation (connect them to administration, other RSEs, researchers, etc)
+
+3) Make it a THING (administrative existance)
+    - Hire RSEs
+    - If you have them already, gather them ("virtual team" or central RSE group)
+    - Offer support to spread good enough software practices in research groups
+    - Train the trainer events are important and needed 
+    - Support formation of RSE groups by providing guidance about practices for forming a new entity in the organisation (group, service, etc) and finding the right integration points
+    - Share organizational knowledge on how get internal funding for RSE-type work
+    
+4) Funding
+    - Inlcude RSEs at the stage of grant proposal writing
+    - Provide central RSE funding to support smaller projects and internal tool developments 
+    - RSE benefits from open ended, non temporary funding - like other techical supports
+    - training and collaboration funding (e.g. EuroCC, EOSC)
```

Comment:
```suggestion
1) Support community involvement and self development
    - Give people room to be active in RSE communities, attend conferences and trainings beyond academic work.

2) Acknowlege RSE work
    - Do not solely focus on authorship when evaluating the scientific output
    - While the contribution of the RSE team/engineer might not be enough for a co-authorship, an acknowledgement in the paper can provide an adequate way to honor the work provided by RSEs
    - Introduce this practice in your organizational guidelines and courses
    - Provide example statements to acknowledge the work behind the scenes (e.g. "Code development and data management were supported by [names of your friendly RSEs or the RSE group].")
 
3) Spread the word - be an RSE ambassador
    - Recognize and highlight RSE (often behind the scenes and non-academic) work
    - Invite RSEs to talk about their work
    - Support RSEs building their own network within the organisation (connect them to administration, other RSEs, researchers, etc)

4) Make it a THING (administrative existance)
    - Hire RSEs
    - If you have them already, gather them ("virtual team" or central RSE group)
    - Offer support to spread good enough software practices in research groups
    - Train the trainer events are important and needed 
    - Support formation of RSE groups by providing guidance about practices for forming a new entity in the organisation (group, service, etc) and finding the right integration points
    - Share organizational knowledge on how get internal funding for RSE-type work
    
5) Funding
    - Inlcude RSEs at the stage of grant proposal writing
    - Provide central RSE funding to support smaller projects and internal tool developments 
    - RSE benefits from open ended, non temporary funding - like other techical supports
    - training and collaboration funding (e.g. EuroCC, EOSC)
```

Please edit as you see fit :)

Reply:
This is a really nice addition, I like it as it is